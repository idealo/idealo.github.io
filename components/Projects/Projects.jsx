import React from "react";
import styles from "./Projects.module.scss";

import ProjectTile from "../../components/ProjectTile/ProjectTile";

const Projects = ({ projectDatas }) => {
  function renderProjectTiles() {
    let tileList = [];

    projectDatas?.forEach((projectData, index) => {
      tileList.push(
        <ProjectTile key={"projectTile" + index} projectData={projectData} />
      );
    });

    return tileList;
  }

  return (
    <section className={styles.projects__container} id="projects">
      <h2 className={`sectionHeading ${styles.projects__heading}`}>Projects</h2>
      <div className={styles.projects__tiles}>{renderProjectTiles()}</div>
    </section>
  );
};
export default Projects;
