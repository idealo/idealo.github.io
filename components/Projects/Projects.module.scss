@import "../../styles/variables.scss";

.projects {
  &__container {
    @extend .centeredContainer;
    padding: 0 40px 30px;
    margin: 0 auto;
    @media screen and (max-width: 500px) {
      margin-top: 10px;
    }
  }

  &__heading {
    border: 2px solid #fff;
  }

  &__tiles {
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
  }
}
