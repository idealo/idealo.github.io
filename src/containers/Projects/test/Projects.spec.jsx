import React from 'react';
import renderer from 'react-test-renderer';
import Projects from '../Projects';

// @todo Added as first nice contribution possibility - doesn't work in github action
xdescribe('Projects component', () => {
    it('should render as expected', () => {
        const renderedComponent = renderer.create(
            <Projects />
        );
        expect(renderedComponent.toJSON()).toMatchSnapshot();
    })
});