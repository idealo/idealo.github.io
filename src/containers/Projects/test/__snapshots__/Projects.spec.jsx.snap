// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component should render as expected 1`] = `
<section
  className="projects__container"
  id="projects"
>
  <h2
    className="sectionHeading projects__heading"
  >
    Projects
  </h2>
  <div
    className="projects__tiles"
  >
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          image-quality-assessment
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_tensorflow.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Convolutional Neural Networks to predict the aesthetic and technical quality of images.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          image-super-resolution
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_tensorflow.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Keras implementation of the Residual Dense Network for super scaling images.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          imageatm
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_tensorflow.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Image classification for everyone.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          cnn-exposed
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_tensorflow.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Interpreting Convolutional Neural Network (CNN) Results.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          nvidia-docker-keras
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_python.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Workflow that shows how to train neural networks on EC2 instances with GPU support and compares training times to CPUs.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          flask-openshift-example
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_python.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Simple Flask example using Docker to deploy on OpenShift 3.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          mongodb-slow-operations-profiler
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_idealo.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        This java web application collects slow operations from a mongoDB system in order to visualize and analyze them.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          terraform-emr-pyspark
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_terraform.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Quickstart PySpark with Anaconda on AWS/EMR using Terraform.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          falcon-prediction-app
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_python.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Simple Machine Learning Web API Example with Falcon.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          logback-redis
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_idealo.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Logback Redis Appender with Pipeline-Support for maximum throughput.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          idealo-orders-api-php-sdk
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_php.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        idealo Direktkauf PHP SDK.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          jenkins-ci
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_idealo.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Minimal example to setup a Jenkins-CI pipeline for data science projects on OpenShift in a couple of minutes.
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          deckard
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_idealo.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Easy-to-use Spring Kafka producers
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="projectTile__container"
    >
      <h3
        className="projectTile__heading"
      >
        <a
          className="projectTile__title"
          target="_blank"
        >
          SageMaker Experiments Explorer
        </a>
        <img
          className="projectTile__logo"
          src="/static/logo_idealo.png"
        />
      </h3>
      <p
        className="projectTile__description"
      >
        Easily explore SageMaker Experiments
      </p>
      <button
        className="projectTile__videoButton"
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-play-circle"
          style={
            Object {
              "fontSize": "20px",
            }
          }
        />
         Video Presentation
      </button>
      <div
        className="projectTile__content"
      >
        <div
          className="gitHubStats__container"
          onClick={[Function]}
        >
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-code-fork"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-star"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
          <div
            className="gitHubStats__stat"
          >
            <p
              className="gitHubStats__icon"
            >
              <span
                aria-hidden={true}
                className="fa fa-users"
              />
            </p>
            <p
              className="gitHubStats__description"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
